redfish/manager.go

```diff
@@ -315,6 +320,7 @@ func (manager *Manager) UnmarshalJSON(b []byte) error {
 		VirtualMedia         common.Link
 		Links                linkReference
 		Actions              actions
+		Oem                  interface{}
```

Comment:
I think common.Links should be used here.

Reply:
I changed to map[string]interface{}. Seems like common.Links have issues to parse sample objects.